src/pages/home/ReportScreen.js

```diff
@@ -175,9 +178,11 @@ class ReportScreen extends React.Component {
     /**
      * @param {SyntheticEvent} e
      */
-    updateViewportOffsetTop(e) {
+    updateViewportValues(e) {
         const viewportOffsetTop = lodashGet(e, 'target.offsetTop', 0);
+        const viewportHeightWithKeyboard = lodashGet(e, 'target.height', 0);
         this.setState({viewportOffsetTop});
+        this.setState({viewportHeightWithKeyboard});
```

Comment:
@shonsirsha Can we have both states set from single this.setState ?

Reply:
Oops, yes ofc!